### PR TITLE
AUA view-online adjustments

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2022-02-04 at 11 11 19 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[AUA eNewsetters](https://trello.com/c/BGn8Nw8g/258-aua-enewsetters)

File: tenants/all/config/core.js [hunk 1]
```diff
@@ -26,8 +26,9 @@ const config = {
   },
   'aua-daily': {
     ...brands.aua,
-    name: 'AUA Daily',
```
Comment (Shinsina) [inline tenants/all/config/core.js:29]: Is there other places this field gets used besides this header location?

Comment (jwade1327) [reply tenants/all/config/core.js:29]: @Shinsina I looked and couldn't find another place where it is using that information

File: packages/common/components/blocks/view-online.marko [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
+import { get } from "@parameter1/base-cms-object-path";
 
 $ const { config, website } = out.global;
 $ const { newsletter } = input;
 $ const newsletterConfig = config.get(newsletter.alias);
+$ const useCustomLinkText = get(newsletterConfig, "customLinkText");
```
Comment (Shinsina) [inline packages/common/components/blocks/view-online.marko:7]: Change this to just customLinkText

File: tenants/all/config/core.js [hunk 3]
```diff
@@ -28,6 +28,8 @@ const config = {
     ...brands.aua,
     name: 'AUA Daily',
     description: 'AUA Daily eNews',
+    customLinkText: 'AUA Daily News Online',
+    showHeaderWebsiteLink: false,
```
Comment (Shinsina) [inline tenants/all/config/core.js:32]: This should always be true, remove this value.

File: packages/common/components/blocks/view-online.marko [hunk 4]
```diff
@@ -16,12 +18,15 @@ $ const showHeaderWebsiteLink = defaultValue(newsletterConfig.showHeaderWebsiteL
             <if(showViewOnline)>
               <a href="@{mv_online_version}@" style="text-decoration: none;color: #62626c;">View in browser</a>
             </if>
-            <if(showViewOnline && showHeaderWebsiteLink)>
+            <if(showViewOnline && showHeaderWebsiteLink || useCustomLinkText)>
```
Comment (Shinsina) [inline packages/common/components/blocks/view-online.marko:21]: I would modify the conditional structure here to something like this:

``` 
      if (showViewOnline && showHeaderWebsiteLink) 
          if (customLinkText) // Display the custom link text
          else // Display website.get("name")
```

**Review by Shinsina — APPROVED** [review]

**Pull request merged**